src/data/reference/hi.json

```diff
@@ -426,7 +500,7 @@
             "returns": "p5 ऑब्जेक्ट"
         },
         "curveTangent": {
-            "description": "बिंदु a, b, c, d के लिए स्थिति t पर वक्र की स्पर्शरेखा का मूल्यांकन करें। पैरामीटर टी 0 और 1 के बीच भिन्न होता है, ए और डी वक्र के बिंदु हैं, बी और सी नियंत्रण बिंदु हैं।",
+            "description": "ए, बी, सी, डी के लिए स्थिति टी पर वक्र के स्पर्शरेखा का मूल्यांकन करता है। पैरामीटर टी 0 और 1 के बीच भिन्न होता है, ए और डी वक्र पर बिंदु होते हैं, और बी और सी नियंत्रण बिंदु होते हैं।",
```

Comment:
Don't translate the function names, variable names, constant names , etc into hi di please !

Reply:
ok